src/components/workflow/SelectBundle.vue

```diff
@@ -0,0 +1,182 @@
+<template>
+<div class="col-lg-7">
+   <modal v-if="workflowSubmission.showSelectBundle" id="selectBundle" @close="workflowSubmission.showSelectBundle = false" class="my-modal">
+      <h3 slot="header">Select Primaryfiles from saved bundles</h3>
+      <div slot="body" class="input-group mb-3">      
+      <div id="accordion" v-if="bundles && bundles.length>0" class="bundles">
+         <div class="card" v-for="(bundle, index) in bundles" v-bind:key="index" >
+            <div class="card-header" id="headingTwo">
+               <h3 class="mb-0">
+                <div class="btn btn-link bundle" :class="{ 'collapsed': visible !== index }" :key="bundle.id" 
+				  	      v-on:click="expandBundle(index)" data-toggle="collapse" data-target="#collapseTwo" 
+					      aria-expanded="visible === index" aria-controls="collapseTwo" :title="[bundle.description]">
+                  <span class="bundle-content">
+                     <svg aria-hidden="true" focusable="false" class="svg-inline dwn-arrow" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 320 512">
+                        <path class="hotlink" d="M285.476 272.971L91.132 467.314c-9.373 9.373-24.569 9.373-33.941 0l-22.667-22.667c-9.357-9.357-9.375-24.522-.04-33.901L188.505 256 34.484 101.255c-9.335-9.379-9.317-24.544.04-33.901l22.667-22.667c9.373-9.373 24.569-9.373 33.941 0L285.475 239.03c9.373 9.372 9.373 24.568.001 33.941z"></path>
+                     </svg>
+                     {{bundle.name}}
+                  </span>
+                  <span class="bundle-content">
+                     {{bundle.createdBy}}
+                  </span>
+                  <button class="btn btn-link float-right" v-on:click="addAllFiles(bundle)" v-bind:disabled=allSelected(bundle) :key="updateSelectedFiles">
+                     <svg class="icon-plus" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 311.5 311.5" style="enable-background:new 0 0 311.5 311.5;" xml:space="preserve">
+                        <path class="circle-stroke" d="M156.8,302c-80.6,0-146.2-65.6-146.2-146.2S76.2,9.6,156.8,9.6S303,75.2,303,155.8S237.4,302,156.8,302z
+                           M156.8,27.9c-70.5,0-127.9,57.4-127.9,127.9s57.4,127.9,127.9,127.9s127.9-57.4,127.9-127.9S227.3,27.9,156.8,27.9z"></path>
+                        <path class="minus-stroke" d="M220.7,164.9H92.8c-5,0-9.1-4.1-9.1-9.1s4.1-9.1,9.1-9.1h127.9c5,0,9.1,4.1,9.1,9.1S225.8,164.9,220.7,164.9z"></path>
+                        <path class="plus-stroke" d="M165.9,91.8v127.9c0,5-4.1,9.1-9.1,9.1s-9.1-4.1-9.1-9.1V91.8c0-5,4.1-9.1,9.1-9.1S165.9,86.8,165.9,91.8z"></path>
+                     </svg>
+                     Add all files
+                  </button>
+                </div>
+               </h3>
+            </div>
+            <div id="collapseTwo" class="collapse" :class="{ 'show' : visible === index }" aria-labelledby="headingTwo" data-parent="#accordion">
```

Comment:
Could this be its own separate component?  Something like "PrimaryFileList"?  Maybe not necessary, but looks reused from SelectFiles.vue

Reply:
i thought about it but didn't bother. in fact the whole accordion could be a generic component. I suspect that Maria might want further UI additions - for ex, the primaryfile label is not unique, maybe we need to add item/collection and such. We can do refactor at that point.